dafni_cli/utils.py

```diff
@@ -70,4 +70,23 @@ def process_date_filter(date_str: str) -> str:
     Returns:
         str: Processed date str to YYYY-MM-DDT00:00:00
     """
-    return dt.strptime(date_str, "%d/%m/%Y").strftime("%Y-%m-%dT%H:%M:%S")
\ No newline at end of file
+    return dt.strptime(date_str, "%d/%m/%Y").strftime("%Y-%m-%dT%H:%M:%S")
+
+
+def argument_confirmation(
+        argument_names: List[str], arguments: List[str], additional_messages: Optional[List[str]]=None
+):
+    """Function to display the arguments and options chosen by the user
+    and ask for confirmation
+
+    Args:
+        argument_names (List[str]): List of the names of the arguments/options for the command
+        arguments (List[str]): List of values chosen for each corresponding argument/option
+        additional_messages (Optional[List[str]]): Other messages to be added after options are listed
+    """
+    for i in range(len(argument_names)):
```

Comment:
I always prefer enumerate in these case:
`for i, value in enumerate(argument_names):`
`    click.echo(f"{value}:{arguments[i]}")`

Reply:
Changed, and made `confirmation_message` an argument for more flexibility.